controller/internal/routes/base_router.go

```diff
@@ -197,8 +197,8 @@ func Create(rc *response.RequestContext, rr response.RequestResponder, sc *gojso
 		rr.RespondWithCouldNotParseBody(err)
 		return
 	}
-
-	il := gojsonschema.NewGoLoader(in)
+	
+	il := gojsonschema.NewBytesLoader(body)
```

Comment:
This is the validation change from go object to JSON I mentioned in the call. Happens 3 times (create, update, patch)